webapp/src/app/admin/users/table/users-table.component.html

```diff
@@ -0,0 +1,152 @@
+    <hlm-th *brnHeaderDef></hlm-th>
+    <hlm-td *brnCellDef="let element">
+      <button hlmBtn variant="ghost" class="h-6 w-6 p-0.5" align="end" [brnMenuTriggerFor]="menu">
+        <hlm-icon class="w-4 h-4" name="lucideMoreHorizontal" />
+      </button>
+
+      <ng-template #menu>
+        <hlm-menu>
+          <hlm-menu-group>
+            <button hlmMenuItem (click)="copyLogin(element)">Copy login</button>
+          </hlm-menu-group>
+          <hlm-menu-separator />
+          <hlm-menu-group>
+            <a hlmMenuItem [href]="element.url" target="_blank">View Github</a>
+            <a hlmMenuItem [routerLink]="'/user/' + element.login">View profile page</a>
+          </hlm-menu-group>
+        </hlm-menu>
+      </ng-template>
+    </hlm-td>
+  </brn-column-def>
+  <div class="flex items-center justify-center p-20 text-muted-foreground" brnNoDataRow>
+    <div class="flex flex-col items-center justify-center gap-2 mt-1">
+      <hlm-icon name="lucideXOctagon" strokeWidth="2px" size="lg" class="text-github-danger-foreground" />
+      <span class="font-semibold text-muted-foreground">No entries found</span>
+    </div>
+  </div>
+</brn-table>
+<div
+  class="flex flex-col justify-between mt-4 sm:flex-row sm:items-center"
+  *brnPaginator="let ctx; totalElements: _totalElements(); pageSize: _pageSize(); onStateChange: _onStateChange"
+>
+  <span class="text-sm text-muted-foreground">{{ _selected().length }} of {{ _totalElements() }} row(s) selected</span>
+  <div class="flex mt-2 sm:mt-0">
+    <brn-select class="inline-block" placeholder="{{ _availablePageSizes[0] }}" [(ngModel)]="_pageSize">
+      <hlm-select-trigger class="inline-flex mr-1 h-9">
+        <hlm-select-value />
+      </hlm-select-trigger>
+      <hlm-select-content>
+        @for (size of _availablePageSizes; track size) {
+          <hlm-option [value]="size">
+            {{ size === 10000 ? 'All' : size }}
+          </hlm-option>
+        }
+      </hlm-select-content>
+    </brn-select>
+
+    <div class="flex space-x-1">
+      <button size="sm" variant="outline" hlmBtn [disabled]="!ctx.decrementable()" (click)="ctx.decrement()">Previous</button>
+      <button size="sm" variant="outline" hlmBtn [disabled]="!ctx.incrementable()" (click)="ctx.increment()">Next</button>
+    </div>
+  </div>
+</div>
+<div class="flex gap-4 mt-4">
+  <brn-select class="inline-block" placeholder="Select a team" [(ngModel)]="_selectedTeam">
+    <hlm-select-trigger class="inline-flex">
+      <hlm-select-value />
+    </hlm-select-trigger>
+    <hlm-select-content class="min-w-[9.5rem]">
+      @for (team of _availableTeams(); track team.id) {
+        <hlm-option [value]="team" class="containerSize">
+          <span class="">{{ team.name }}</span>
+        </hlm-option>
+      }
+    </hlm-select-content>
+  </brn-select>
+  <button hlmBtn variant="secondary" [disabled]="_selected().length === 0 || !_selectedTeam" (click)="addTeamToSelected()">Add team</button>
```

Comment:
adding team doesnt not directly show up. need to reload first

Reply:
I improved the validation logic for it in 9c19d1e, please let me know in case this still occurs now.